sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -203,6 +203,12 @@ impl Transport for CW310 {
             usb.pin_set_state(CW310::PIN_SW_STRAP0, true)?;
             usb.pin_set_state(CW310::PIN_SW_STRAP1, true)?;
             usb.pin_set_state(CW310::PIN_SW_STRAP2, true)?;
+            usb.pin_set_state(CW310::PIN_TAP_STRAP1, true)?;
```

Comment:
i think whats most important here is setting sensible defaults states for the tap straps to match [here](https://github.com/lowRISC/opentitan/pull/12633#discussion_r871938097), which this does. nice catch!

Reply:
I think it's actually above where we clear PIN_TAP_STRAP0 so the SPI bus gets enabled when we also clear PIN_TAP_STRAP1, the rest is just to bring this in-line with the tested behavior until this is the tested behavior.